CMakeLists.txt

```diff
@@ -904,7 +904,7 @@ set(CTEST_CUDA_LAUNCHER_OPTIONS
       "" CACHE STRING
       "Options to pass to the CTEST_MPI_LAUNCHER to select GPU nodes")
 
-if( BUILD_TESTING )
+if( BUILD_TESTING AND "${CMAKE_SOURCE_DIR}" STREQUAL "${PROJECT_SOURCE_DIR}" )
```

Comment:
If you are in "BUILD_TESTING" mode and you integrate parsec as a submodule, I'd argue that you want to run the testers on PaRSEC as well.